src/large_pages/node_large_page.cc

```diff
@@ -106,18 +106,18 @@ namespace node {
 namespace {
 
 struct text_region {
-  char* from;
-  char* to;
-  int   total_hugepages;
-  bool  found_text_region;
+  char* from = nullptr;
+  char* to = nullptr;
+  bool found_text_region = false;
 };
 
 static const size_t hps = 2L * 1024 * 1024;
 
 template <typename... Args>
-inline void Debug(Args&&... args) {
+inline void Debug(std::string fmt, Args&&... args) {
   node::Debug(&per_process::enabled_debug_list,
               DebugCategory::HUGEPAGES,
+              (std::string("Hugepages info: ") + fmt).c_str(),
               std::forward<Args>(args)...);
```

Comment:
Given that this is already tied to the `DebugCategory::HUGEPAGES`, is the prefix even necessary? I've been considering whether `node::Debug` should automatically prefix statements with the category name.

Reply:
Well, if several categories are given, debug statements from them might become interleaved. We could certainly push prefixing with the category into `node::Debug`, but that's beyond the scope of this PR, IMO.